geomstats/geometry/skew_symmetric_matrices.py

```diff
@@ -9,7 +9,7 @@
 from geomstats.geometry.matrices import Matrices
 
 
-TOLERANCE = 1e-12
+TOLERANCE = 1e-8
```

Comment:
Maybe name this `TOL`? we have `tol`, `atol`, `rtol` as naming in the code. Unless it is the tolerance for a particular function, in this case `***_TOL` could be more explicit?

Side remark: numpy default tolerances are `rtol=1e-5, atol=1e-8`, for example: 
https://docs.scipy.org/doc/numpy/reference/generated/numpy.isclose.html. I wonder if we could factorize our tolerance levels somewhere?

Reply:
it would be helpful indeed